### PR TITLE
Translation process of a file project In Crowdin 

### PR DESCRIPTION
**Pull request by Rubadel**
Localization of the Arabic language, for  -Local and National Publishers in Indonesia and the impact in the open Since srt- File

### CODE REVIEW

File: docs/RubaAdel_review.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
+**Crowding مساهمتي من خلال منصة**
```
Comment (BatoolMM) [inline docs/RubaAdel_review.md]: This sentence is aligned left to the right - it should be right to the left, can you fix it?

**Review by BatoolMM — CHANGES_REQUESTED** [review]

**Review by Rubadel — COMMENTED** [review]
@BatoolMM  Done 🌸

File: docs/RubaAdel_review.md (hunk 1 above)
Comment (Rubadel) [reply docs/RubaAdel_review.md]: I fixed it

**Comment by BatoolMM** [comment]
@all-contributors please add @Rubadel for translation and documentation

**Comment by BatoolMM** [comment]
Thank you so much @Rubadel!

**Pull request merged**